### DISCUSSION TITLE
make system collections available by API (ReST or GraphQL)

### DISCUSSION THREAD

**Discussion by ilseva** [directus/directus#23194] [Feature Requests]
### Summary

It would be nice to have access to system collections (eg: directus_collections) by GET or SEARCH /items API 

### Basic Example

curl -X GET "https://your-directus-instance.com/system/collections?filter[group][_eq]=example_group" \
     -H "Authorization: Bearer YOUR_ACCESS_TOKEN"

### Motivation

We are using Directus as headless CMS and it would be useful to have access to the data model to design custom UI

### Detailed Design

enhance the logic on isSystemCollection method on packages/system-data/src/collections/index.ts considering the user logged in.
if he is an admin or has a role with the permission to read the specific system collection, return true.

### Must Have:
- Read through API system collections

### Drawbacks

no reason

### Alternatives

I didn't find alternatives

### Adoption Strategy

no breaking change

**Comment by itszeeshan** [comment]
it is there already :)
https://docs.directus.io/reference/system/collections.html

**Comment by ilseva** [comment]
Sorry,
wrong example

```shell
curl --request GET \
  --url 'http://localhost:8055/items/directus_collections?filter[group][_eq]=organization' \
  --header 'Authorization: Bearer <token>'
```
response is 403
```json
{
	"errors": [
		{
			"message": "You don't have permission to access this.",
			"extensions": {
				"code": "FORBIDDEN"
			}
		}
	]
}
```
Both in ReST API and GraphQL, the "system" collections of directus are not "readable".

**Reply by hanneskuettner** [reply]
The collections are available at the `/collections` endpoint. Not `/items/collections` or `/items/directus_collections`. System collections are handled differently and are available at a root level, without the `directus_` prefix.

**Reply by ilseva** [reply]
Thanks, but `/collections` doesn't support query parameters... 
If I design a hierarchical structure based on `group` field of `directus_collections` and I need to find all the collections in a group I don't have an API to do this.

**Comment by ilseva** [comment]
linked to https://github.com/directus/directus/discussions/23249. You can close one.
